distributed/tests/test_scheduler.py

```diff
@@ -2339,101 +2339,103 @@ def assert_memory(scheduler_or_workerstate, attr: str, min_, max_, timeout=10):
             raise TimeoutError(
                 f"Expected {min_} MiB <= {attr} <= {max_} MiB; got:\n{minfo!r}"
             )
-        sleep(0.1)
+        await asyncio.sleep(0.1)
 
 
 # This test is heavily influenced by hard-to-control factors such as memory management
 # by the Python interpreter and the OS, so it occasionally glitches
 @pytest.mark.flaky(reruns=3, reruns_delay=5)
 # ~33s runtime, or distributed.memory.recent_to_old_time + 3s
 @pytest.mark.slow
```

Comment:
I would assume test runtime is impacted by this (to be better). Is it still "slow"?

Reply:
test runtime does not change if you manually initialise the workers or if you let a fixture initialise them for you.